From e3ee8f69bb47e5fe44d86e658ab92ac4ddab6008 Mon Sep 17 00:00:00 2001
From: Zhenxiao Luo <zhenxiao@cloudera.com>
Date: Wed, 20 Jun 2012 15:11:16 -0700
Subject: [PATCH 037/144] CLOUDERA-BUILD. Make cdh4-0.9.0 build/test for both MR1 and MR2

Reason: CLOUDERA-BUILD
Author: Zhenxiao Luo
Ref: CDH-5265

The build command has been simplified to:

Build Cmd:
$ant very-clean tar

Run Test in MR1 Mode:
$ant test -Dhadoop.mr.test.rev=20

Run Test in MR2 Mode:
$ant test
---
 build.properties |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/build.properties b/src/build.properties
index f33eccd..6c7da0b 100644
--- a/src/build.properties
+++ b/src/build.properties
@@ -39,7 +39,7 @@ hadoop.mirror2=http://archive.cloudera.com/hive-deps
 hadoop.mr.rev=23
 hadoop.test.version=${hadoop.version}
 hadoop.mr.test.rev=${hadoop.mr.rev}
-hadoop.mr1.test.version=${hadoop.test.version}
+hadoop.mr1.test.version=2.0.0-mr1-cdh4.0.0-SNAPSHOT
 
 build.dir.hive=${hive.root}/build
 build.dir.hadoop=${build.dir.hive}/hadoopcore
-- 
1.7.0.4

